.circleci/config.yml

```diff
@@ -101,10 +101,9 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/
-      - aws-cli/install
       - aws-cli/configure:
-          aws-access-key-id: AWS_ACCESS_KEY_ID
-          aws-secret-access-key: AWS_SECRET_ACCESS_KEY
+          role-arn: "arn:aws:iam::702835727665:role/circleci-public-repos"
```

Comment:
Job is failing with `Cannot find a definition for command named aws-cli/configure`